spec/acceptance/api/internal/services_api_spec.rb

```diff
@@ -78,6 +78,17 @@
           expect(response_json['service']['state']).to eq 'suspended'
         end
       end
+      context 'with an service that has no state in db' do
+        let(:id) { otherid }
+        let(:storage) { ThreeScale::Backend::Storage.instance }
+        example 'Get Service by ID should return inactive state' do
+          storage.del ThreeScale::Backend::Service.storage_key(id, 'state')
```

Comment:
I think that this kind of test belongs in the unit tests/specs for `Service` - this is an API spec and the storage and its representation are implementation details that should not appear here. I know this escaped my radar with the other specs here, but let's have it moved to the unit tests.